kratos/python/add_mesh_to_python.cpp

```diff
@@ -469,6 +475,7 @@ void  AddMeshToPython(pybind11::module& m)
     .def("Calculate", &ElementCalculateInterface<Matrix >)
     .def("CalculateMassMatrix", &ElementCalculateMassMatrix)
     .def("CalculateDampingMatrix", &ElementCalculateDampingMatrix)
+    .def("CalculateLumpedMassVector", &ElementCalculateLumpedMassVector)
```

Comment:
this is minor, but do we need this to be exposed?

Reply:
I am not sure. I added for sake of consistency.

Reply:
I was thinking the same. But considering that the other elemental contributions are exposed, I'd keep them for the sake of consistency (they might eventually be useful for debugging or for more academic purposes).